src/tool_getparam.c

```diff
@@ -593,8 +592,10 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
       noflagged = TRUE;
     }
 
+    /* search for the accompanying internal short letter */
     for(j = 0; j < sizeof(aliases)/sizeof(aliases[0]); j++) {
-      if(curl_strnequal(aliases[j].lname, word, fnam)) {
+      /* only compare the prefix in order to guess the remainder */
```

Comment:
It doesn't "guess" the remainder, it is happy with a prefix as long as it is a unique hit. I think "guess" gives the reader the wrong idea.

Reply:
ACK